### PR TITLE
Update data.toml to include new auto responses

### PR DESCRIPTION
**Pull request by Rairof**

### CODE REVIEW

File: src/data.toml [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+ios_app_icons = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively breaks being able to change app icons.\nYou need to sign the application with a local dev cert instead if you want this functionality.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
 addon_404 = "You are not supposed to visit the url directly since it does not serve anything that can be rendered. Copy the link and use it to install the add-on."
+not_support_channel = "This is not the support channel, please move to https://discord.com/channels/1196075698301968455/1210652540757090315."
+cloudsync_not_working = "You might be using a outdated version of cloud sync as older versions and vendetta version doesn't work anymore-\n- check pins and install latest cloud sync (https://bunny.nexpid.xyz/cloud-sync)\n- If that doesn't work, uninstall and reinstall https://angelix1.github.io/MP/angel/antied/"
```
Comment (nexpid) [inline src/data.toml:19]:
```suggestion
cloudsync_not_working = "You might be using an outdated version of Cloud Sync, as older versions don't work anymore.\n1. Make sure you have the latest version of Cloud Sync (available in the pins) available\n2. If that doesn't work, and you have **antied** installed, uninstall and reinstall it (https://angelix1.github.io/MP/angel/antied/)"
```

**Review by nexpid — CHANGES_REQUESTED** [review]

**Comment by Rairof** [comment]
do not merge, adding more auto responses

**Comment by Rairof** [comment]
@castdrian review changes pls

**Comment by Rairof** [comment]
Added PR to pyoncord auto support fork

**Pull request closed without merging**